### ISSUE TITLE
Query.setLimit() only works if there is a where clause

### ISSUE THREAD

**Issue by rafarc** [BraintagsGmbH/vertx-pojo-mapper#34]
Recently I noticed that query.setLimit() isn't working. I checked the source code to see what was happening, and saw this:

*(SqlExpression.java)*
```java
private void appendLimitClause(StringBuilder complete) {
    if (whereClause.length() > 0) {
      if (limit > 0) {
        complete.append(" LIMIT ").append(limit);
      }
      if (offset > 0) {
        complete.append(" OFFSET ").append(offset);
      }
    }
  }
```

So, the limit will be applied only when there is a *where* clause. In my code base, I have many queries without a *where* clause.
Any reasoning behind this decision?

**Comment by remi128** [comment]
nope - i'll fix it

**Comment by remi128** [comment]
fixed; added test in 1.3.2-SNAPSHOT
